Foundation/NSPathUtilities.swift

```diff
@@ -616,12 +628,22 @@ public func NSHomeDirectoryForUser(user: String?) -> String? {
 }
 
 public func NSUserName() -> String {
```

Comment:
Looking at this further; it might be reasonable to actually just call out to CFCopyUserName